src/controller.rs

```diff
@@ -56,7 +56,6 @@ pub struct Context {
 
 #[derive(Clone, Default)]
 pub struct Secrets {
-    pub wasmcloud_cluster_seed: String,
     pub nats_creds: Option<String>,
```

Comment:
🤔  should this secret be optional in crates/types/src/v1alpha1/wasmcloud_host_config.rs too?
only data left is `nats_creds` which is already optional

Reply:
heads up took a stab at it in #68